libexec/rc/rc.conf

```
@@ -353,7 +353,7 @@ amd_map_program="NO"		# Can be set to "ypcat -k amd.master"
 autofs_enable="NO"		# Run autofs daemons.
 automount_flags=""		# Flags to automount(8) (if autofs enabled).
 automountd_flags=""		# Flags to automountd(8) (if autofs enabled).
-autounmountd_flags=""		# Flags to autounmountd(8) (if autofs enabled).
+autounmountd_flags="-t1 -r1"	# Flags to autounmountd(8) (if autofs enabled).
```

```suggestion
autounmountd_flags=""	# Flags to autounmountd(8) (if autofs enabled).
```
Reset back to FreeBSD defaults. Probably not needed on TrueOS.